client/src/components/Home.jsx

```diff
@@ -97,6 +97,15 @@ const Home = ({isMobile, isLoggedIn, isLoading, setIsLoggedIn, userEmail, shared
     //   .catch(err => console.error(err));
   }
 
+  const addCategory = (category) => {
+    console.log('Add Category: ', category);
+    axios.post('/category', {  data: category })
```

Comment:
will also need the params for userEmail in this as well for the query

Reply:
That makes sense/I agree. I've got to start running the data through from the beginning, and I'll fix bugs as I work towards the back. Just setting up some basic chunks to work with.

Reply:
Just updated locally. Can push if you'd like.

Reply:
(& see my notes in slack. I can pull this request if you'd like)